packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -233,6 +241,7 @@ def convert_value_to_str(value: object) -> str:
             service_class=service["serviceClass"],
             description_url=service["descriptionUrl"],
             provided_interfaces=service["providedInterfaces"],
+            version=version,
```

Comment:
`MeasurementInfo` and `ServiceInfo` expect a `str` but you are passing an `Optional[str]`.

```
ni_measurement_plugin_sdk_service/measurement/service.py:229: error: Argument "version" to "MeasurementInfo" has incompatible type "Optional[str]"; expected "str"  [arg-type]
ni_measurement_plugin_sdk_service/measurement/service.py:244: error: Argument "version" to "ServiceInfo" has incompatible type "Optional[str]"; expected "str"  [arg-type]
Found 2 errors in 1 file (checked 204 source files)
```

Some ways to handle this:
- Use the default argument `version: str = ""` and rely on "falsiness" instead of checking for None.
- Add a fallback case like `if version is None: version = ""`. I'm not sure if mypy will treat this as a type narrowing, though.
- `version="" if version is None else version,`